### DISCUSSION TITLE
Google Play Keystore change

### DISCUSSION THREAD

**Discussion by olavomello** [expo/expo#17618] [Archived: Questions]
I changed the Keystore and follow the support instruction bellow :

----
... generate a new key:
        
    keytool -genkeypair -alias upload -keyalg RSA -keysize 2048 -validity 9125 -keystore keystore.jks

This key must be a 2048 bit RSA key and have 25-year validity.
Export the certificate for that key to PEM format:
        
    keytool -export -rfc -alias upload -file upload_certificate.pem -keystore keystore.jks 

... Reply to this email and attach the **upload_certificate.pem** file.

---

**A )** Generating APK I'm not receive an alert installing directly in device. ( one problem solved ).
**B )** Publishing the Bundle to Play Store, everything goes fine but uploading the .aab file to Google Play Store I received the error :
 **+ Incorrect key SHA1**
 To try to solve I uploaded new credentials do **Expo Credentials** but I cant due to "Push Key File" alert informing invalid file. ( keystore.jks - generated by command informed from Google Support ).

Can anyone help me to solve this ?

**Comment by luizuk** [comment]
I'm with the same problem. Someone can help?
